scripts/backend.py

```diff
@@ -169,6 +238,312 @@ def page_not_found(error):
     return send_file(in_http_dir("404.html"))
 
 
+# streaming stuff begins
+# streaming overrides cyberia? (configure this in DEFAULT_LIVESTREAM_INFO)
+
+@app.route("/schedule.json", methods=["GET"])
+def schedule():
+    dt = datetime.today()
+    start = dt - timedelta(days=dt.weekday())
+    end = start + timedelta(days=6)
+    week_of = "{} - {}".format(start.strftime('%b %d'), end.strftime('%b %d'))
+    sched_info = {'week_of': week_of}
+    sched_path = os.path.join(db.SAVEDATA_PATH, 'schedule.txt')
+    if os.path.exists(sched_path):
+        with open(sched_path, 'r', encoding='utf-8') as sched_file:
+            sched_lines = sched_file.read()
+            sched_lines = sched_lines.split('\n')
+            sched_lines = [line for line in sched_lines if not line.startswith('//')]
+            sched_info['dailies'] = sched_lines
+    resp = make_response(json.dumps(sched_info), 200)
+    resp.headers["Content-Type"] = "application/json"
+    return resp
+
+
+def start_streaming():
+    # right now, this immediately stop the stream if you call actually_start_streaming instead
+    # this can wait for current track to finish playing if you uncomment below instead 
+    actually_start_streaming()
+    return
```

Comment:
Similarly, this code can be removed as people will start streaming whenever they do, not necessarily when a song ends.